plugins/periskop/README.md

```diff
@@ -0,0 +1,34 @@
+# periskop
+
+[Periskop](https://periskop.io/) is a pull-based, language agnostic exception aggregator for microservice environments.
+
+![periskop-logo](https://i.imgur.com/z8BLePO.png)
+
+### `PeriskopErrorsTable`
+
+The Periskop Backstage Plugin exposes an entity tab component named `PeriskopErrorsTable`. Each of the entries in the table will direct you to the error details in your deployed Periskop instance location.
```

Comment:
not really a "tab component" as such right?

may want to mention some installation steps, like what rough JSX to add in your catalog files in `packages/app`

Reply:
Done, thanks